test/integration/node-specific/client_close.test.ts

```diff
@@ -453,29 +459,143 @@ describe.skip('MongoClient.close() Integration', () => {
   });
 
   describe('ClientSession (Implicit)', () => {
+    let idleSessionsBeforeClose;
+    let idleSessionsAfterClose;
+    let client;
+    let utilClient;
+    let session;
+
+    const metadata: MongoDBMetadataUI = {
+      requires: {
+        topology: ['replicaset', 'sharded'],
+        mongodb: '>=4.2'
+      }
+    };
+
+    beforeEach(async function () {
+      client = this.configuration.newClient();
+      utilClient = this.configuration.newClient();
+      await client.connect();
+      await client
+        .db('db')
+        .collection('collection')
+        ?.drop()
+        .catch(() => null);
+      const collection = await client.db('db').createCollection('collection');
+      session = client.startSession({ explicit: false });
+      session.startTransaction();
+      await collection.insertOne({ x: 1 }, { session });
+
+      const opBefore = await utilClient.db().admin().command({ currentOp: 1 });
+      idleSessionsBeforeClose = opBefore.inprog.filter(s => s.type === 'idleSession');
+
+      await client.close();
+
+      const opAfter = await utilClient.db().admin().command({ currentOp: 1 });
+      idleSessionsAfterClose = opAfter.inprog.filter(s => s.type === 'idleSession');
+
+      await utilClient.close();
+    });
+
+    afterEach(async function () {
+      await utilClient?.close();
+      await session?.endSession();
+      await client?.close();
+    });
+
     describe('Server resource: LSID/ServerSession', () => {
       describe('after a clientSession is implicitly created and used', () => {
-        it.skip('the server-side ServerSession is cleaned up by client.close()', async function () {});
+        it(
+          'the server-side ServerSession is cleaned up by client.close()',
+          metadata,
+          async function () {
+            expect(idleSessionsBeforeClose).to.not.be.empty;
+            expect(idleSessionsAfterClose).to.be.empty;
+          }
+        );
       });
     });
 
     describe('Server resource: Transactions', () => {
       describe('after a clientSession is implicitly created and used', () => {
-        it.skip('the server-side transaction is cleaned up by client.close()', async function () {});
+        it(
+          'the server-side transaction is cleaned up by client.close()',
+          metadata,
+          async function () {
+            expect(idleSessionsBeforeClose[0].transaction.txnNumber).to.not.null;
+            expect(idleSessionsAfterClose).to.be.empty;
+          }
+        );
       });
```

Comment:
This whole code block doesn't make sense - you can't have an implicit session with a transaction.  Right?